.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Ion-test-driver issue.
+assignee: cheqianh
+labels: bug
+---
+Ion-test-driver complained about behavior changed for the commit {{ sha }} that created by `{{ payload.sender.login }}`.
```

Comment:
"complained about behavior changed" -> "complained that behavior changed"
"that created by" -> "created by"